2-dom-manipulation/form.js

```diff
@@ -0,0 +1,65 @@
+let form = document.querySelector('form');
+
+function structureIf(nameCondition){
+    if (nameCondition.value == '') {
+        console.log(`${nameCondition.id}: It's need value`);
+        nameCondition.classList.add('required');
+        return "";
+    } else {
+        nameCondition.classList.remove("required");
+        return nameCondition.value;
+    }
+}
+
+function validEmail(){
+    if (/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/.test(Email.value)) {
```

Comment:
you can save the RegEx in a const.
`const EMAIL_REGEX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/`